.circleci/config.yml

```diff
@@ -1743,3 +1709,20 @@ jobs:
       - run:
           name: All Tests Passed
           command: echo 'whew - everything passed!'
+
+  check-mmi-trigger:
+    executor: node-browsers-medium
+    steps:
+      - checkout
+      - run:
+          name: Check for MMI Team Label or Reviewer
+          command: |
+            chmod +x .circleci/scripts/check_mmi_trigger.sh
```

Comment:
The file permissions are checked into the repository; you don't need to make it executable every run, you can make it executable once and commit it.